src/packages/fetch-engine/src/download.ts

```diff
@@ -218,10 +218,11 @@ function getCollectiveBar(options: DownloadOptions): {
   finishBar: () => void
   setProgress: (sourcePath: string) => (progress: number) => void
 } {
+  const hasNodeAPI = 'libquery-engine' in options.binaries
```

Comment:
Is it possible this does not work on Windows @williamluke4?

Reply:
Don't think so as if you are using NodeAPI then `libquery-engine` will be present in `options.binaries`. Why do you think that this would work on windows?

Reply:
No this should still work

Reply:
See Slack thread for further details and screenshots.